### ISSUE TITLE
qvm-start-daemon - audiovm - keyboard layout

### ISSUE THREAD

**Issue by neowutran** [QubesOS/qubes-issues#8109]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### The problem you're addressing (if any)

When using a audiovm, the system will complain that the audiovm is not allowed to override the keyboard layout. 
Modifying the keyboard layout is normally only related to guivm, not audiovm. 

### The solution you'd like

I believe the issue is here:

https://github.com/QubesOS/qubes-core-admin-client/blob/main/qubesadmin/tools/qvm_start_daemon.py#L800
https://github.com/QubesOS/qubes-core-admin-client/blob/main/qubesadmin/tools/qvm_start_daemon.py#L792

The same actions are executed for both guivm and audiovm. 
This is not a behavior wanted for the audiovm. 
Maybe could add a check like that https://github.com/QubesOS/qubes-core-admin-client/blob/main/qubesadmin/tools/qvm_start_daemon.py#L741 to only execute thoses actions if the VM ask for guivm 

### The value to a user, and who that user might be

No useless warning, so not thinking that something don't work

**Comment by ben-grande** [comment]
@neowutran is this still an issue? I see that https://github.com/QubesOS/qubes-core-admin-client/pull/237 was closed and was unsure with the replies.

**Comment by neowutran** [comment]
No,  this issue is indeed solved with marmarek suggested policy rule, and added this rule in the audiovm guide back then
